cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -204,20 +200,38 @@ public void createCollection(String dbName, String collectionName, long collecti
                 default:
              }
             options = new IndexOptions().name("cyg_raw_opt");
-            db.getCollection(collectionName).createIndex(keys, options);
+            createIndex(db, collectionName, keys, options);
         } catch (Exception e) {
             LOGGER.warn("Error in collection " + collectionName + " creating index ex=" + e.getMessage());
         } // try catch
```

Comment:
Should the try/catch be removed (given that createIndex() will deal with the exception internally)?

Reply:
Fixed in 1458daf

Reply:
It seems the try/catch block is still there... or maybe I'm missing something?

Reply:
Ok, will remove try/catch.

Reply:
Fixed in b6045a0